### PR TITLE
feat(table): add fullscreen loading state demo

### LINKED ISSUE #7342: Feat(Table): Full screen demo for loading state

### PR DESCRIPTION
**Pull request by jenny-s51**
**What**: Closes #7342

**Convenience link to demo:** https://patternfly-react-pr-7370.surge.sh/components/table/react-demos/loading/

### CODE REVIEW

**Review by mcarrano — APPROVED** [review]
This looks great @jenny-s51 . Please disregard my comment on your other PR then as I didn't realize you were making these demo changes separately.

**Review by mmenestr — APPROVED** [review]

**Review by tlabaj — APPROVED** [review]

File: packages/react-table/src/demos/Table.md [hunk 1]
```diff
@@ -3054,7 +3054,12 @@ class LoadingStateDemo extends React.Component {
             props: { colSpan: 8 },
             title: (
               <Bullseye>
-                <Spinner size="xl" aria-labelledby="loading-table-demo" />
+                <EmptyState>
+                  <EmptyStateIcon variant="container" component={Spinner} />
+                  <Title size="lg" headingLevel="h4">
```
Comment (mcoker) [inline packages/react-table/src/demos/Table.md:3059]: Looks like this should be an `<h2>` since the only other heading on the page is an `<h1>`

**Review by mcoker — CHANGES_REQUESTED** [review]
LGTM! Just a couple of small nits.

**Review by jenny-s51 — COMMENTED** [review]
Thank you @mcoker - updated based on your feedback!

**Review by mcoker — APPROVED** [review]
lgtm!!

**Pull request merged**